scripts/kibana_proxy.py

```diff
@@ -53,46 +57,56 @@ class KibanaProxy:
 
     def __init__(self, options) -> None:
         self.options = options
+        self.docker = docker.from_env()
+
+    def create_container(self, image: str, command=None, **kwargs) -> Container:
```

Comment:
*nit* : taking the `image` string and splitting it into a `image_name` and `tag`, then setting the `image` to an object is a little strange.

I would rename the `image: str` parameter to something else, unfortunately i cant find a fully qualified name for the `repository/image:tag`, perhaps `image_alias`?

https://docs.docker.com/glossary/

Reply:
Good point. After reading a little, I found that I've been using the wrong terminology. Given `lmenezes/cerebro:0.8.5`,

* `lmenezes/cerebro` is the repository
* `0.8.5` is the tag
* `lmenezes/cerebro:0.8.5` has no "official" name; some refer to it as the fully-qualified image name (FQIN) but the Python API refers to it only as "name"

I've pushed an update that uses this terminology.